fenix/app/src/test/java/org/mozilla/fenix/telemetry/TelemetryMiddlewareTest.kt

```diff
@@ -86,6 +86,7 @@ class TelemetryMiddlewareTest {
         val engine: Engine = mockk()
         every { engine.enableExtensionProcessSpawning() } just runs
         every { engine.disableExtensionProcessSpawning() } just runs
+        every { engine.getSupportedTranslationLanguages(any(), any()) } just runs
```

Comment:
In response to [CI issue here](https://firefoxci.taskcluster-artifacts.net/WIEwqhImSiaR2bmFoGw9mA/0/public/reports/test/testFenixDebugUnitTest/classes/org.mozilla.fenix.translations.TranslationsDialogBindingTest.html#WHEN%20translate%20action%20success%20is%20sent%20to%20the%20browserStore%20THEN%20update%20translation%20dialog%20store%20based%20on%20operation).